bazel/ci/keep_sorted.sh.in

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+###### script header ######
+
+lib=$(realpath @@BASE_LIB@@) || exit 1
+stat "${lib}" >> /dev/null || exit 1
+
+# shellcheck source=../sh/lib.bash
+if ! source "${lib}"; then
+  echo "Error: could not find import"
+  exit 1
+fi
+
+keep_sorted=$(realpath @@KEEP_SORTED@@)
+stat "${keep_sorted}" >> /dev/null
+
+cd "${BUILD_WORKSPACE_DIRECTORY}"
+
+###### script body ######
+
+find . -not -path "./.git/*" -type f | sort | xargs "${keep_sorted}" --mode fix
```

Comment:
This is probably already done incorrectly throughout the codebase, but I wonder if we should add hermetic find, sort and xargs. We did sometimes see differences between macOS and Linux. Feel free to ignore for now. Maybe worth an extra ticket / PR.

Reply:
We already provide coreutils from nixpkgs, doing the same for findutils wouldn't be too hard, but is out of scope for this PR imo.

Reply:
I think `sort` and `xargs` are specified sufficiently so that this invocation should be portable.

`find` on the other hand just surprised me: `-not` is apparently non-standard, but both BSD and GNU support it. That should be ok for now.